core/controllers/acl_decorators.py

```diff
@@ -38,13 +40,31 @@
 from core.domain import user_services
 from core.platform import models
 import feconf
-import python_utils
 
 current_user_services = models.Registry.import_current_user_services()
 
 (suggestion_models,) = models.Registry.import_models([models.NAMES.suggestion])
 
 
+def redirect_based_on_return_type(
```

Comment:
Should this be prefixed with an underscore (since it seems to be an helper function internal to this module)?

(Do the same for all such functions -- in general prefer making things private where possible. It helps with managing complexity by not exposing so many functions to the outside.)

Reply:
Done.